contrib/debian/netdata-plugin-perf.postinst

```diff
@@ -8,7 +8,9 @@ case "$1" in
     chmod 0750 /usr/libexec/netdata/plugins.d/perf.plugin
 
     if capsh --supports=cap_perfmon 2>/dev/null; then
-        setcap cap_perfmon+ep /usr/libexec/netdata/plugins.d/perf.plugin
+        if ! setcap cap_perfmon+ep /usr/libexec/netdata/plugins.d/perf.plugin; then
+            echo "Failed to set cap_perfmon capability, trying fallback. . ."
```

Comment:
```suggestion
            chmod -f 4750 /usr/libexec/netdata/plugins.d/perf.plugin
```

Reply:
there is no fallback in this branch

Reply:
You have this fallback to 4750 when doing `setcap cap_sys_admin+ep`, why not here too?

Reply:
perfmon -> fallback to cap_sys_admin -> fallback to setuid

Reply:
You don't have fallback if https://github.com/netdata/netdata/blob/091a4790dcd62428a030037086e3a090df42318d/contrib/debian/netdata-plugin-perf.postinst#L11C14-L11C34 fails

Reply:
w8 yes I got a logic error,  thanks but still the fallback of the check you mentioned above shouldn't be the chmod 4750

Reply:
please check again